javascript-data-model/order-history.js

```diff
@@ -0,0 +1,52 @@
+var orderHistory = [
+  {
+    name: 'Javascript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    orderNumber: '114-3941689-8772232',
+    orderPlaced: 'August 4, 2020',
+    orderDelivered: 'Aug 8, 2020',
+    returnUntil: 'Sep 7, 2020',
+    recipient: 'JS masher',
+    price: 31.55
+  },
+  {
+    name: 'The Timeless way of building',
+    author: 'Alexander, Christopher',
+    orderNumber: '113-9984268-1280257',
+    orderPlaced: 'July 19, 2020',
+    orderDelivered: 'Jul 20, 2020',
+    returnUntil: 'Aug 19, 2020',
+    recipient: 'JS masher',
+    price: 41.33
+  },
+  {
+    name: 'GameCube Controller Adapter. SSB switch GameCube Adapter for WII, etc. Support turbo and vibration features.',
+    author: null,
+    orderNumber: '114-2875557-9059409',
+    orderPlaced: 'July 4, 2020',
+    orderDelivered: 'Jul 7, 2020',
+    returnUntil: 'Aug 5, 2020',
+    recipient: 'JS masher',
+    price: 15.98
+  },
+  {
+    orderNumber: '113-2883177-2648248',
+    orderPlaced: 'July 3, 2020',
+    orderDelivered: 'July 5, 2020',
+    returnUntil: 'Aug 4, 2020',
+    recipient: 'JS masher',
+    total: 138.83,
+    items: [
+      {
+        name: 'GameCube controller- SSB. Edition',
+        author: null,
+        price: 94.95
+      },
+      {
+        name: 'The art of Sql',
+        author: 'Faroult, Stephane',
+        price: 33.99
+      }
+    ]
```

Comment:
The structure of this order's object does not match the structure of your other objects, which will lead to bugs if you tried to use this data in an actual application. The shape of your data in each order object needs to be consistent.